btm/pom.xml

```diff
@@ -16,9 +16,10 @@
             <artifactId>jta</artifactId>
         </dependency>
         <dependency>
-            <groupId>org.apache.geronimo.specs</groupId>
-            <artifactId>geronimo-jms_1.1_spec</artifactId>
+            <groupId>javax.jms</groupId>
+            <artifactId>javax.jms-api</artifactId>
```

Comment:
is this compatible with 3.x in the case we need to make changes there?

Reply:
yes, is compatible, si the same lib